pkg/mbedtls/include/riot_mbedtls_config.h

```diff
@@ -229,6 +229,47 @@ extern "C" {
 #endif
 /** @endcond */
 
+#if MBEDTLS_ENABLED == 1
```

Comment:
Please expose (only necessary) symbols via Kconfig, as requested by @miri64 and @aabadie in a conversation starting somewhere [here](https://github.com/RIOT-OS/RIOT/pull/15671#issuecomment-747633130)